packages/google_maps_flutter/google_maps_flutter/ios/Classes/GoogleMapHeatmapController.m

```diff
@@ -0,0 +1,159 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "GoogleMapHeatmapController.h"
+#import "JsonConversions.h"
+@import GoogleMapsUtils;
+
+@implementation FLTGoogleMapHeatmapController {
+  GMUHeatmapTileLayer *_heatmap;
+  GMSMapView *_mapView;
+}
+- (instancetype)initWithHeatmap:(GMUHeatmapTileLayer *)heatmap mapView:(GMSMapView *)mapView {
+  self = [super init];
+  if (self) {
+    _heatmap = heatmap;
+    _mapView = mapView;
+
+    // Workaround for weird heatmap rendering compared to other platforms
+    // Context:
+    // https://github.com/googlemaps/google-maps-ios-utils/issues/143#issuecomment-343850029
```

Comment:
I didn't get the choices of the zoom intensity values based on the comment in this link. Could you explain?
In general, setting an undocumented arbitrary values seems error prone to me.

Reply:
I chose a min of 0 because that's the absolute minimum. I chose 21 as the max because I think that's the max zoom possible on a google map. I honestly don't really know why/how this works, but I do know that the iOS heatmaps render very differently than Android/web and other people that have tried to implement this have had weird issues such as the heatmap basically being invisible because of assumptions made in the rendering. Setting these values gave me close results to what heatmaps look like on other platforms.

Reply:
I did a quick search and couldn't find anything about it. If it is an undocumented workaround, I don't feel comfortable hard code this in the OBJC implementation. Is there a way to make these 2 values configurable from dart public API so users can provide these workaround themselves? Are these properties available on other platforms as well?

Reply:
Yes I could make these configurable from flutter. No other platforms don't seem to have them.

Reply:
Done